composer.json

```diff
@@ -45,6 +42,13 @@
     "squizlabs/php_codesniffer": "^3.4.2",
     "roave/security-advisories": "dev-master"
   },
+  "config": {
+    "optimize-autoloader": true,
+    "preferred-install": "dist",
+    "sort-packages": true
+  },
+  "minimum-stability": "dev",
```

Comment:
What’s up with this?

Reply:
https://getcomposer.org/doc/articles/autoloader-optimization.md#optimization-level-1-class-map-generation + package sorting when doing `composer require ...` or even `composer update` after adding a package by hand (a-z)